doc/develop/extending.rst

```diff
@@ -179,34 +179,32 @@ right-click and select *Git Bash here*. Then type::
 Creating the Odemis environment
 -------------------------------
 
-Install `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ and
+Install `Miniconda <https://docs.anaconda.com/miniconda/install/#quick-command-line-install>`_ and
 pick the right architecture for your computer (most likely 64-bit).
 Odemis is validated on Python 3.6, but Anaconda typically comes with a newer version of Python (eg, 3.9)
 The following instructions use the fact that Anaconda supports multiple versions of Python 
 simultaneously to install Python 3.6.
 
 Setup
 """""
-Open the *Anaconda prompt* and type::
+Open *PowerShell* and type::
```

Comment:
Could you update the same way the instructions which start line 296. No need to check them on a Windows. I assume that if you change them similarly to the changes here, it will already be a lot more correct that what currently is written!